File: java/gazelle/testdata/grpc/src/main/java/com/example/hello/hello.java
package com.example.hello;

import com.example.hello.proto.DeleteBookRequest;
import com.example.hello.proto.HelloProto;

public class Hello {
    public static void main(String[] args) {
        DeleteBookRequest req = DeleteBookRequest.newBuilder().setName("book 1").build();
        System.out.println("Hello, World!");
    }
}

File: java/test/com/github/bazel_contrib/contrib_rules_jvm/javaparser/generators/workspace/com/gazelle/kotlin/javaparser/generators/Hello.kt
package workspace.com.gazelle.kotlin.javaparser.generators

import com.gazelle.java.javaparser.generators.DeleteBookRequest
import com.gazelle.java.javaparser.generators.HelloProto
import com.google.common.primitives.Ints

class Hello {
  fun useImports() {
    val req: DeleteBookRequest? = null
    val proto: HelloProto? = null
    val bytes = Ints.BYTES
    // Touch the values so tools consider them used
    if (bytes >= 0) {
      println("$req $proto $bytes")
    }
  }
}
